website/server/controllers/api-v3/challenges.js

```diff
@@ -257,6 +257,14 @@ api.createChallenge = {
       privacy: group.privacy,
     };
 
+    res.analytics.track('create challenge', {
+      uuid: user._id,
+      hitType: 'event',
+      category: 'behavior',
+      groupName: group.privacy === 'private' ? null : group.name,
+      groupType: group._id === TAVERN_ID ? 'tavern' : group.type,
+    });
```

Comment:
Should this record the group ID? The name alone isn't fully reliable since it can change plus its annoying to search for if we want to find the guild.
Can it have the challenge ID too? That could be useful occasionally for troubleshooting.

Reply:
Sure! I tend to think of these bits from the behavioral data side rather than troubleshooting utility, so that's a good perspective to get.